docs/cache/develop/sdks/nodejs/config-and-error-handling.mdx

```diff
@@ -0,0 +1,53 @@
+---
+sidebar_position: 2
+sidebar_label: Config and Error Handling
+title: Observability for TypeScript and Node.js
```

Comment:
`title` and `description` are the fields used for social cards when a link is shared via slack/social media. We always need to make sure we're updating these and adding `keywords` for SEO purposes. 

`title`: Observe your Momento Cache metrics in TypeScript and Node.js
`description`: Learn how to add observability tools into your TypeScript and Node.js code for viewing Momento Cache metrics.
`keywords`:
  - momento
  - cache
  - serverless
  - observability
  - sdk